packages/elements/src/components/FileInput/index.tsx

```diff
@@ -35,17 +35,23 @@ export const handleChangeCroppedImage = ({
   inputFile,
 }: {
   field: FieldInputProps<string>
-  croppedImage?: string
+  croppedImage?: string | null
   setFileName: React.Dispatch<string>
   inputFile: React.RefObject<HTMLInputElement>
 }) => () => {
-  field.onChange({ target: { value: croppedImage ?? '', name: field.name } })
+  // when croppedImage is not set or when first mount
+  if (croppedImage === undefined || croppedImage === null) {
```

Comment:
I think can change `if (!croppedImage) {`

Reply:
We can't, because `croppedImage` can be `''` and we need to handle that case separately